src/primary-process/index.ts

```diff
@@ -233,6 +233,26 @@ export const offloadReceipt = async (
   receivedReceiptCount++ // Increment the counter for each receipt received
   receiptLoadTraker++ // Increment the receipt load tracker
   let verificationResult: ReceiptVerificationResult
+
+  // Check if offloading is disabled globally or for global modifications
+  if (
+    config.disableOffloadReceipt ||
+    (config.disableOffloadReceiptForGlobalModification && receipt.globalModification)
+  ) {
+    mainProcessReceiptTracker++
+    if (config.workerProcessesDebugLog) console.log('Verifying on the main program', txId, timestamp)
+    verificationResult = await verifyArchiverReceipt(receipt, requiredSignatures)
```

Comment:
With these flags we still want the verifyArchiverReceipt to happen but on the main thread. This will avoid the verification all together.

Reply:
Ignore this